### PR TITLE
Introduce Thoth Test and required changes

### PR DESCRIPTION
**Pull request by harshad16**
Introduce Thoth Test and required changes
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Introduce one new group and modified older group
- thoth: all members
- thoth-devops: only devops team

modified namespace interaction based on these groups.

### CODE REVIEW

File: cluster-scope/base/namespaces/thoth-test-core/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
```
Comment (HumairAK) [inline cluster-scope/base/namespaces/thoth-test-core/namespace.yaml:3]: Did you mean to add this namespace to the zero cluster [kustomization](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/zero/kustomization.yaml) ?

Comment (harshad16) [reply cluster-scope/base/namespaces/thoth-test-core/namespace.yaml:3]: thank you, I couldn't find the kustomize file before :see_no_evil: , I have updated it.

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]

**Pull request merged**